### DISCUSSION TITLE
How to use a colour from the pallette?

### DISCUSSION THREAD

**Discussion by erlendnagel** [paulscottrobson/neo6502-firmware#281] [Q&A] [answered]
I was busy converting a mandelbrot program hoping to use the 256 colours of the neo6502 , but unfortunately I was not able to pass any value higher then 15 to ink. Does the neo6502 really only have 16 colours available from a pallette of 256? And are those 16 colours shared by sprites, tiles, characters and drawing primitives?

**Comment by paulscottrobson** [comment, marked as answer]
Nope.  It has a maximum of 64 different colours, with a palette of 256. In its default structure this is used to split in two bitplanes of 4 bits each ; one for static text/graphics, the other for sprites. However, if you don't use sprites then the drawing function which take ink 0-255 values, as does the palette command, should make all 64 variants available.

Though actually there's no reason, I think, why the palette system couldn't support 256 different colours.
